satpy/readers/mirs.py

```diff
@@ -122,7 +122,8 @@ def read_atms_limb_correction_coefficients(fn):
 
         # coeff locations (indexes to put the future coefficients in)
         locations = [int(x.strip()) for x in next(coeff_str).split(" ") if x]
-        assert(len(locations) == nchx)
+        if (len(locations) != nchx):
```

Comment:
Since this PR is specifically about style issues, could you remove the extra parentheses here?

Reply:
```suggestion
        if len(locations) != nchx:
```